suites/pacific/cephadm/tier1_ssl_dashboard_port.yaml

```diff
@@ -304,3 +306,29 @@ tests:
         verify: true
       destroy-cluster: false
       abort-on-fail: true
+  - test:
+      name: Place a host under maintenance using cephadm
+      desc: Use orchestrator to place a host in maintenance mode, verify its status.
+      module: test_host.py
+      polarion-id: CEPH-83573799
+      config:
+        service: host
+        command: enter
+        verify: true
+        args:
+          node: node2
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Exit a host from maintenance using cephadm
+      desc: Use orchestrator to exit maintenance mode on a host, verify its status.
+      module: test_host.py
+      polarion-id: CEPH-83573799
+      config:
+        service: host
+        command: exit
+        verify: true
+        args:
+          node: node2
+      destroy-cluster: false
+      abort-on-fail: true
```

Comment:
It is a good practice to end with a newline character.

Reply:
Done.